src/python_interpreter.rs

```diff
@@ -29,79 +30,155 @@ print(json.dumps({
+                    .get(1)
+                    .unwrap()
+                    .as_str()
+                    .parse::<usize>()
+                    .context(context)?;
+                let minor = capture
+                    .get(2)
+                    .unwrap()
+                    .as_str()
+                    .parse::<usize>()
+                    .context(context)?;
+                if !versions_found.contains(&(major, minor)) {
+                    let pointer_width = capture
+                        .get(3)
+                        .unwrap()
+                        .as_str()
+                        .parse::<usize>()
+                        .context(context)?;
+
+                    if windows_interpreter_no_build(
+                        major,
+                        minor,
+                        target.pointer_width(),
+                        pointer_width,
+                    ) {
+                        continue;
+                    }
+
+                    let version = format!("-{}.{}-{}", major, minor, pointer_width);
+
+                    let output = Command::new("py")
+                        .args(&[&version, "-c", code])
+                        .output()
+                        .unwrap();
+                    let path = str::from_utf8(&output.stdout).unwrap().trim();
+                    if !output.status.success() || path.trim().is_empty() {
+                        bail!("Couldn't determine the path to python for `py {}`", version);
+                    }
+                    interpreter.push(path.to_string());
+                    versions_found.insert((major, minor));
+                }
             }
+        }
+    }
 
-            // There can be 32-bit installations on a 64-bit machine, but we can't link
-            // those for 64-bit targets
-            if pointer_width != target.pointer_width() {
-                println!(
-                    "{}.{} is installed as {}-bit, while the target is {}-bit. Skipping.",
-                    major,
-                    minor,
-                    pointer_width,
-                    target.pointer_width()
-                );
-                continue;
+    // Conda environments are also supported on windows
+    let conda_info = Command::new("conda").arg("info").arg("-e").output();
+    if let Ok(output) = conda_info {
+        let lines = str::from_utf8(&output.stdout).unwrap().lines();
+        let re = Regex::new(r"(\w|\\|:|-)+$").unwrap();
+        let mut paths = vec![];
+        for i in lines {
+            if !i.starts_with('#') {
+                if let Some(capture) = re.captures(&i) {
+                    paths.push(String::from(&capture[0]));
+                }
             }
+        }
 
-            let version = format!("-{}.{}-{}", major, minor, pointer_width);
-
-            let output = Command::new("py")
-                .args(&[&version, "-c", code])
+        for path in paths {
+            let executable = format!(r"{}\python", path);
```

Comment:
Nit: Use [Path::join](https://doc.rust-lang.org/nightly/std/path/struct.Path.html#method.join) instead of format